data/social.tsx

```diff
@@ -0,0 +1,25 @@
+type Social = {
```

Comment:
The file can be renamed to `socialMediaLinks.js` to make it more understandable at first glance.